### ISSUE TITLE
Currently does not support array index references example item.0.name

### ISSUE THREAD

**Issue by CarlTierney** [DanieleScipioni/Mustache.UWP#2]
The library does not support the array index notation for mustache

**Comment by DanieleScipioni** [comment]
I'm evaluating to support this syntax. After a review of the mustache specifications (https://github.com/mustache/spec) it seems to me that is not part of it.

Can you anyway specify which is your requirement?
If your requirement is to access only the first element than you can takes as example mustache demo at https://mustache.github.io/#demo. 
If you want an access to the collection by index, what about to expose it as a `Dictionary<string, string>` where the keys are the indexes?

**Comment by CarlTierney** [comment]
You are correct, it is not in the spec.    https://github.com/mustache/mustache/issues/222  I made the code change already and its in the pull request.  It does work on the javascript version (mustache.js).  My product's wysiwyg editor is providing validation that it does work to the client due to mustache.js's support that does not work server side without that capability in the C# version.  I stopped using your released released version and just pulled the source into my project for now with the change that is in the pull request.
